docs/components/modeler/web-modeler/connectors/available-connectors/slack.md

```diff
@@ -0,0 +1,92 @@
+---
+id: slack
+title: Slack connector
+description: Working with Slack connector
+---
+
+The **Slack connector** allows you to send messages to channels or users in your [Slack](https://slack.com) workspace from your BPMN process.
+
+## Prerequisites
+
+To use the Slack connector, a Slack app must be registered with the Slack workspace you would like to send messages to. A respective OAuth token needs to be configured as a secret in your cluster. Follow [these steps in the appendix](#appendix) to learn how to set this up.
+
+## Create a Slack connector task
+
+To use a **Slack connector** in your process, either change the type of an existing task using the **change type** context menu, or create a new connector task by using the **append** context menu. Follow [our guide on using connectors](../use-connectors.md) to learn more.
+
+## Make your Slack connector executable
+
+To make the **Slack connector** executable, fill out the mandatory fields highlighted in red in the properties panel:
+
+![slack connector red properties](../img/connectors-slack-red-properties.png)
+
+1. Set **Authentication** to your Slack OAuth token, which is stored as a secret. For example 'secrets.SLACK_OAUTH_TOKEN`
+2. Set **Channel/User Name** to either the **channel** or **user** you want to send the message to
+   1. A **channel** is specified by an unique identifier starting with an `#` (example: `#myChannel`)
```

Comment:
Very minor comment, update text to:
"A **channel** is specified by a unique identifier starting with an `#` (example: `#myChannel`)"

Reply:
:+1: